packages/composites/augur/src/dataProviders/sportsdataio.ts

```diff
@@ -142,7 +250,21 @@ const getScores = async (
       }))
     }
     case 'ncaa-fb': {
-      return (filtered as CFBGames[]).map(event => ({
+      const input = {
+        id,
+        data: {
+          sport,
+          endpoint: 'scores',
+          season: '2021', // TODO: Find a source for this information.
+        },
+      }
+
+      const response = await exec(input, context)
+      const filtered = (response.result as { GlobalGameID: number }[]).filter(
+        (event) => event.GlobalGameID != 0,
```

Comment:
also

Reply:
That only showed up changed due to formatting. 🤷